--- /dev/null
+++ BUILD.bazel
@@ -0,0 +1,7 @@
+package(default_visibility = ["//visibility:public"])
+
+ros2_interface_library(
+    name = 'test_interface_files_msgs',
+    rule_suffix = ['msg/Nested.msg', 'msg/Arrays.msg', 'msg/BoundedSequences.msg', 'msg/MultiNested.msg', 'msg/Empty.msg', 'msg/WStrings.msg', 'msg/BasicTypes.msg', 'msg/UnboundedSequences.msg', 'msg/Constants.msg', 'msg/BoundedPlainSequences.msg', 'msg/Strings.msg', 'msg/Defaults.msg'],
+)
+
